optuna/visualization/matplotlib/_pareto_front.py

```diff
@@ -0,0 +1,237 @@
+from typing import List
+from typing import Optional
+
+import optuna
+from optuna._experimental import experimental
+from optuna.study import Study
+from optuna.trial import FrozenTrial
+from optuna.trial import TrialState
+from optuna.visualization.matplotlib._matplotlib_imports import _imports
+
+
+if _imports.is_successful():
+    from optuna.visualization.matplotlib._matplotlib_imports import Axes
+    from optuna.visualization.matplotlib._matplotlib_imports import plt
+
+_logger = optuna.logging.get_logger(__name__)
+
+
+@experimental("2.7.0")
```

Comment:
Could you update this version number?
```suggestion
@experimental("2.8.0")
```

Reply:
Nice catch! Thanks!